### PR TITLE
Fix/customize flutter build

### PR DESCRIPTION
**Pull request by johannesvedder**
- Several QoL CLI changes and fixes
  - remove whitespaces from token generation
  - fix studyu status
  - studyu delete now works with multiselect
  - supabase secrets can be created by cli even if supabase .env did not exist
  - style fixes
- StudyU images now rely directly on the flutter repo
- Melos bootstrap will not run in parallel inside the Dockerfile
- Enable icon tree shaking for all web builds
---
- [ ] See https://github.com/hpi-studyu/studyu/pull/551#discussion_r1431687845

### CODE REVIEW

File: docker/studyu/Dockerfile [hunk 1]
```diff
@@ -1,41 +1,56 @@
...
 
 # Install melos
 RUN dart pub global activate melos
 
-# Copy repository files
-COPY ./ ./
-
 # Clean repository without failing
-RUN melos clean
+RUN melos --verbose clean
```
Comment (johannesvedder) [inline docker/studyu/Dockerfile:34]: All --verbose should be removed before merge

**Review by hig-dev — APPROVED** [review]

File: docker/studyu/Dockerfile (hunk 1 above)
Comment (hig-dev) [reply docker/studyu/Dockerfile:34]: Thanks for the hint.

```suggestion
RUN melos clean
```

File: docker/studyu/Dockerfile [hunk 2]
```diff
@@ -1,41 +1,56 @@
...
 
 # Setup workspace
 RUN dart pub get
-RUN melos bootstrap
+RUN melos --verbose bootstrap
```
Comment (hig-dev) [inline docker/studyu/Dockerfile:38]:
```suggestion
RUN melos bootstrap
```

File: docker/studyu/Dockerfile [hunk 3]
```diff
@@ -1,41 +1,56 @@
...
 
 # Build app
-RUN melos run build:web:$FLUTTER_APP_FOLDER
+RUN melos --verbose run build:web:$FLUTTER_APP_FOLDER
```
Comment (hig-dev) [inline docker/studyu/Dockerfile:41]:
```suggestion
RUN melos run build:web:$FLUTTER_APP_FOLDER
```

**Review by hig-dev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**